app/Ticket.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Ticket extends Model
+{	
+	protected $fillable = [
+		'title', 'content', 'slug', 'status', 'user_id',
```

Comment:
mỗi tham số ở 1 dòng
```suggestion
		'title', 
		'content', 
		'slug', 
		'status', 
		'user_id',
```